backend/capellacollab/projects/routes.py

```diff
@@ -182,6 +188,14 @@ def delete_project(
     crud.delete_project(db, project)
 
 
+def _delete_all_pipelines(db: orm.Session, project: models.DatabaseProject):
+    pipelines: list[backups_models.DatabaseBackup] = []
+    for model in project.models:
+        pipelines.extend(backups_crud.get_pipelines_for_tool_model(db, model))
+    for pipeline in pipelines:
+        backups_crud.delete_pipeline(db, pipeline)
```

Comment:
A Database deletion is not sufficient. This would leave behind the cronjob in the cluster, which is not referenced anymore!

Reply:
Good catch missed that. Deletion should not work similar to. the usual pipeline deletion